<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Telefonica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>HNLogo.png</normaloff>HNLogo.png</iconset>
  </property>
  <widget class="QPushButton" name="GetLocalRegistryButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Get local registry info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetRemoteRegistryWinRMButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Retrieve remote info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewProjectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Networking options (WMI)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Local computer options</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ShowCurrentProject">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenProjectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Project </string>
   </property>
  </widget>
  <widget class="QLabel" name="ProjectNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Project name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProjectFileNameText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProjectCSVFileNameText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProjectJSONFileNameText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ProjectFileNameLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Project file  </string>
   </property>
  </widget>
  <widget class="QLabel" name="ProjectCSVFileNameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>CSV file</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProjectJSONFileNameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>JSON file</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ProjectJSONFileNameLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>USB devices detected:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProjectJSONFileNameLabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Computer Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="USBDetectedText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ComputerNameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadComputerListButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Load list of computers</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ComputerFileListShow">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="DrawButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Plot Project</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginUserInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordUserInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Domain credentials</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetLocalComputer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetRemoteOutput">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DrawButtonSingleCSV">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Plot single CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>HN.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="AboutButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSaveCSV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save output to CSV file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
    </font>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Telefonica</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Plot options</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
